pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "builddsl"
-version = "1.0.0"
+version = "1.0.1"
```

Comment:
The bump happens when I run `slap release` :)

```suggestion
version = "1.0.0"
```

Reply:
ups, sorry - I did `slap release` on my fork repo =D ...I'll revert it now